1-exercises/B-object-arrays/exercise-2.js

```diff
@@ -16,10 +16,9 @@ let store = [
 
 let money = 1.5;
 
-let thingsICanBuy =
-  /* todo */
+let thingsICanBuy = store
+  .filter((item) => item.price <= money)
+  .map((item) => item.item);
```

Comment:
Good use of filter and map - maybe use a different variable name as item.item could be confusing.